devenv/config.toml

```diff
@@ -0,0 +1,6 @@
+[python]
+version = '3.8.18'
+darwin_x86_64 = 'https://github.com/indygreg/python-build-standalone/releases/download/20231002/cpython-3.8.18+20231002-x86_64-apple-darwin-install_only.tar.gz'
+darwin_x86_64_sha256 = 'fcf04532e644644213977242cd724fe5e84c0a5ac92ae038e07f1b01b474fca3'
+darwin_arm64 = 'https://github.com/indygreg/python-build-standalone/releases/download/20231002/cpython-3.8.18+20231002-aarch64-apple-darwin-install_only.tar.gz'
+darwin_arm64_sha256 = '1825b1f7220bc93ff143f2e70b5c6a79c6469e0eeb40824e07a7277f59aabfda'
```

Comment:
This format requires some string splitting/joining to use it programattically. If we can preserve the structure of the data, that's better:

`.sentry-devenv.json`
```
{
  "python": {
    "version": "3.8.18",
    "release": "20231002",
    "url": "https://github.com/indygreg/python-build-standalone/releases/download/{release}/cpython-{version}+{release}-{arch}-{kernel}-install_only.tar.gz",
    "sha256": {
      "kernel": {
        "apple-darwin": {
          "arch": {
            "x86_64": "fcf04532e644644213977242cd724fe5e84c0a5ac92ae038e07f1b01b474fca3",
            "aarch64": "1825b1f7220bc93ff143f2e70b5c6a79c6469e0eeb40824e07a7277f59aabfda"
          }
        }
      },
      "kernel": {
        "unknown-linux-gnu": {
          "arch": {
            "x86_64": "26247302bc8e9083a43ce9e8dd94905b40d464745b1603041f7bc9a93c65d05",
            "aarch64": "2e84fc53f4e90e11963281c5c871f593abcb24fc796a50337fa516be99af02fb"
          }
        }
      }
    }
  }
}
```

It could be yaml but then devenv will need to add a dependency for that.
I'd then add a `devenv bump-python` subcommand to help update this file.

Reply:
I don't feel like this is better -- it still has string formatting and is harder to just copy paste urls

Reply:
just gonna go with ini